tests/cargo-kani/stubbing-double-extern-path/harness/expected

```diff
@@ -1 +1 @@
-error[E0391]: cycle detected when optimizing MIR for `crate_a::assert_true`
```

Comment:
FYI, this test checks that there is a cycle so the error printed here depends on which method we visit first. But the cycle is correctly detected no matter what.